dump.md

```diff
@@ -106,26 +106,34 @@ As I sort links, they may not fit in any of the categories I'm currently sorting
-- OPERAS - [COESO: Collaborative Engagement on Societal Issues](https://www.operas-eu.org/projects/coeso/)
-- OPERAS - [VERA: Enabling collaborative research with and for society](https://operas-eu.org/services/vera/) and [access to the service](https://vera.operas-eu.org/en/home)
-- Conference: [Connect, collaborate, create](https://ccc.sciencesconf.org/)
-- PHDontrack - [Pre-registration](https://www.phdontrack.net/open-science/preregistration/index.html)
-- [OSFregistries](https://osf.io/registries/) for research pre-registration
-- [Humanities Commons](https://hcommons.org/) for humanities research pre-registration
-- [Octopus](https://science-octopus.org/) - micro publications
-- JOTE - [Journal of Trial and Error](https://archive.jtrialerror.com/)
+- :arrow_forward:  OLS - [Github for collaboration](https://youtu.be/7Oq81K27NmU) Video su come usare GitHub
+- :closed_lock_with_key: 	:department_store: PARTHENOS - [Virtual research environment](https://parthenos.d4science.org/web/parthenos_vre) Service platform
+
+### Citizen Science
+- :small_red_triangle_down: ORION [Inspiring stories](https://www.orion-openscience.eu/public/2021-07/ORION_inspiring_stories_booklet_2021_0.pdf) on co-creation, citizen science etc
+- 	:department_store: Platform on Citizen science in Europe [European citizen science initiative](https://eu-citizen.science/)
+- 	:department_store: :school: PARTHENOS: course about how to do citizen science in Humanities and Digital Arts [Citizen science and the humanities](https://training.parthenos-project.eu/sample-page/citizen-science-in-the-digital-arts-and-humanities/)
+- :information_desk_person:  :it: An example of Citizen Science in Italy (Maremma) [Citizen Science Italia](https://www.museonaturalemaremma.it/csi/) and [its creation announcement](https://www.nnb.isprambiente.it/it/nasce-lassociazione-nazionale-citizen-science-italia-ets)
+-  :information_desk_person: Example of Citizen Science (Dutch) [Moving forward together with Open Science](https://www.rathenau.nl/en/inclusive-science/moving-forward-together-open-science)
+- :speech_balloon: [Meaningful participation in Citizen Science, a strategy](https://www.linkedin.com/pulse/meaningful-participation-citizens-research-strategy-frederike-schmitz/)
+- :department_store: :school: Cos4EOSC - [Co-creation for EOSC](https://cos4cloud-eosc.eu/) Focus on how to do co-creation
+-  :department_store: :school: OPERAS - [COESO: Collaborative Engagement on Societal Issues](https://www.operas-eu.org/projects/coeso/)
+- :department_store: OPERAS - Focus on co-creation and citizen science in Social Sciences and Humanities[VERA: Enabling collaborative research with and for society](https://operas-eu.org/services/vera/) and [access to the service](https://vera.operas-eu.org/en/home)  
+- :information_desk_person: Registration of a conference on citizen science: [Connect, collaborate, create](https://ccc.sciencesconf.org/)
+
+###  :page_facing_up: Pre-registration and pre-print  
```

Comment:
Perhaps an introductory facet (or, even better, links) about what pre-registration is and why it is useful? Maybe a link to a relevant bit of the Open Science section of the DSK?